modules/serverless-logic-job-service-eventing-api-config.adoc

```diff
@@ -0,0 +1,135 @@
+// Module included in the following assemblies:
+//
+// * serverless/serverless-logic/serverless-logic-job-service.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="serverless-logic-job-service-eventing-api-config_{context}"]
+= Eventing API configurations
+
+The Job service supports a Cloud Event based API for creating and deleting jobs. This API is used for deployment scenarios where event based communication between the workflow runtime and the Job service is needed. You can use either the Knative Eventing system or the Kafka messaging system to transport these events.
+
+[id="serverless-logic-job-service-knative-eventing-system_{context}"]
+== Knative Eventing system
+
+By default, the Job service Eventing API is configured to work with the Knative Eventing system. Without any additional configuration, it can receive CloudEvents through Knative Eventing to manage jobs. You must still set up your Knative Eventing environment to ensure proper delivery of these events to the Job service. 
```

Comment:
You could rephrase last part of the sentence for clarity:

```suggestion
By default, the Job service Eventing API is configured to work with the Knative Eventing system. Without any additional configuration, it can receive CloudEvents through Knative Eventing to manage jobs. Ensure that your Knative Eventing environment is properly configured to guarantee the correct delivery of CloudEvents to the Job service. 
```